### PR TITLE
chore (litmus-portal): Added argo workflow representation using dagre d3 graph

### PR DESCRIPTION
**Pull request by arkajyotiMukherjee**
## Proposed changes

- Argo workflow representation using dagre d3
- Subscription fixes
- removed unnecessary re-rendering
- refactoring of few components
- renaming to appropriate names for few components

Summarize your changes here to communicate with the maintainers and make sure to put the link of that issue
fixes issue #1881 

## Types of changes

What types of changes does your code introduce to Litmus? Put an `x` in the boxes that apply
- [x] New feature (non-breaking change which adds functionality)
- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices applies)

## Checklist

Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.
- [ ] I have read the [CONTRIBUTING](https://github.com/litmuschaos/litmus/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the commit for DCO to be passed.
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works (if appropriate)
- [ ] I have added necessary documentation (if appropriate)

## Dependency
- Please add the links to the dependent PR need to be merged before this (if any).

### CODE REVIEW

**Review by amityt — DISMISSED** [review]
LGTM

**Review by gdsoumya — DISMISSED** [review]
LGTM

**Review by S-ayanide — DISMISSED** [review]

File: litmus-portal/backend/auth/go.sum [hunk 1]
```diff
@@ -4,6 +4,7 @@ github.com/davecgh/go-spew v1.1.1 h1:vj9j/u1bqnvCEfJOwUhtlOARqs3+rkHYY13jYWTU97c
 github.com/davecgh/go-spew v1.1.1/go.mod h1:J7Y8YcW2NihsgmVo/mv3lAwl/skON4iLHjSsI+c5H38=
```
Comment (rajdas98) [inline litmus-portal/backend/auth/go.sum:4]: Can you remove these changes

**Comment by rajdas98** [comment]
Can you resolve the merge conflicts?

File: litmus-portal/backend/auth/go.sum (hunk 1 above)
Comment (arkajyotiMukherjee) [reply litmus-portal/backend/auth/go.sum:4]: My origin has been updated using the update PR button....now if I do a reset and remove the changes, then the changes that were added on top of my two commits will also we removed

**Review by gdsoumya — APPROVED** [review]
LGTM

**Review by S-ayanide — APPROVED** [review]

**Review by amityt — APPROVED** [review]
LGTM

**Review by rajdas98 — APPROVED** [review]

**Pull request merged**